common.yml

```diff
@@ -65,6 +65,10 @@
         with_sequence: 0-{{ (oc_instackenv_content.nodes|length - 1) }}
         register: host_list
 
+      - name: pull badfish image
```

Comment:
can you move this task to bootstrap.yml  playbook as common.yml  playbook contains only tasks that are needed by all the tags